libraries/chain/include/eosio/chain/abi_serializer.hpp

```diff
@@ -442,18 +445,14 @@ namespace impl {
       }
 
       /**
-       * overload of to_variant_object for packed_transaction
+       * overload of to_variant_object for packed_transaction_v0
        *
        * This matches the FC_REFLECT for this type, but this is provided to allow extracting the contents of ptrx.transaction
-       * @tparam Resolver
-       * @param act
-       * @param resolver
-       * @return
        */
       template<typename Resolver>
-      static void add( mutable_variant_object &out, const char* name, const packed_transaction& ptrx, Resolver resolver, abi_traverse_context& ctx )
+      static void add( mutable_variant_object &out, const char* name, const packed_transaction_v0& ptrx, Resolver resolver, abi_traverse_context& ctx )
       {
-         static_assert(fc::reflector<packed_transaction>::total_member_count == 4);
+         static_assert(fc::reflector<packed_transaction_v0>::total_member_count == 4);
          auto h = ctx.enter_scope();
          mutable_variant_object mvo;
          auto trx = ptrx.get_transaction();
```

Comment:
`const auto&`

Reply:
Updated `signed_transaction`, `transaction`, `transaction_header` to disallow implicit copy. Removed copies we do not need.